sql/2021/javascript/web_components_specs.sql

```diff
@@ -0,0 +1,30 @@
+SELECT
+  client,
+  percentile,
+  APPROX_QUANTILES(custom_elements, 1000)[
+    OFFSET(percentile * 10)] AS custom_elements,
+  APPROX_QUANTILES(shadow_roots, 1000)[
+    OFFSET(percentile * 10)] AS shadow_roots,
+  APPROX_QUANTILES(template, 1000)[
+    OFFSET(percentile * 10)] AS template
+FROM (
+  SELECT
+    _TABLE_SUFFIX AS client,
+    CAST(JSON_EXTRACT(JSON_EXTRACT_SCALAR(payload,
+          '$._javascript'),
+        '$.web_component_specs.custom_elements') AS INT64) AS custom_elements,
```

Comment:
IIUC [these custom metrics](https://github.com/HTTPArchive/legacy.httparchive.org/blob/master/custom_metrics/javascript.js#L54) are arrays of node names, but here you're casting them to `INT64`. Are you looking to calculate the distribution of the number of custom elements per page? Something else?